react/src/lib/components/SubsurfaceViewer/layers/axes2d/axes2DLayer.ts

```diff
@@ -58,6 +58,7 @@ export interface Axes2DLayerProps<D> extends ExtendedLayerProps<D> {
     labelFontSize?: number;
     fontFamily?: string;
     axisColor?: Color;
+    backGroundColor?: Color;
```

Comment:
Suggest `backgroundColor`

Reply:
Fixed. And also made the text sharper and more readable.